web/src/main/ui/app/components/flows-new/models/step.model.ts

```diff
@@ -1,16 +1,51 @@
-import { Options } from './step-options.model';
+import { IngestionOptions } from './ingestions-options.model';
+import { MappingOptions } from './mapping-options.model';
+import { MasteringOptions } from './mastering-options.model';
+import { CustomOptions } from './custom-options.model';
+
 
 export class Step {
```

Comment:
Please create a private empty constructor so no one could create Step without factory methods. This may cause a number of errors across the app (which is a good thing). Please test the build from scratch.